DataFormats/L1Trigger/src/Muon.cc

```diff
@@ -93,6 +93,23 @@ l1t::Muon::Muon( const PolarLorentzVector& p4,
   
 }
 
+l1t::Muon::Muon( const l1t::Muon& muon)
+{
```

Comment:
Hi Sven,
does this also copy the variables of the L1Candidate?

Reply:
good point, I would need to look into that

Reply:
@rekovic what can you suggest here? I am missing a copy statement of the parent object?